docs/markdown/Reference-tables.md

```diff
@@ -251,7 +251,8 @@ which are supported by GCC, Clang, and other compilers.
 | warning                  |
 | warn_unused_result       |
 | weak                     |
-| weakreaf                 |
+| weakref                  |
+| retain                   |
```

Comment:
We should mark this as new in 0.62.0, I think there's other examples of this. I think this table is alphabetically sorted as well.

Reply:
Changes made. Let me know if it's up-to-spec and I'll rebase.

Reply:
looks good. I can just squash when I commit, so don't worry about rebasing (saves CI time)

Reply:
Sure thing.